source/includes/steps-install-shell-linux-rpm.yaml

```
@@ -31,30 +31,41 @@ content: |
 
             You can also download the ``.rpm`` files directly from the
             `MongoDB repository <https://repo.mongodb.org/yum/redhat/>`__.
-            Downloads are organized by Red Hat or CentOS version (e.g.
-            ``8``), then by MongoDB
-            :manual:`release version </reference/versioning>` (e.g.
-            ``{+mdb-version+}``), then by architecture (e.g. ``x86_64``).
+            Downloads are organized in the following order:
+            
+            1. Red Hat or CentOS version (for example, ``8``)
+
+            #. MongoDB edition (for example, ``mongodb-enterprise``)
+
+            #. MongoDB :manual:`release version </reference/versioning>`
+               (for example, ``{+mdb-version+}``)
+
+            #. Architecture (for example, ``x86_64``)
 
         - id: aws2-tab
           name: Amazon Linux 2
           content: |
 
             .. code-block:: shell
 
-               [mongodb-org-5.0]
+               [mongodb-org-{+mdb-version+}]
                name=MongoDB Repository
-               baseurl=https://repo.mongodb.org/yum/amazon/2/mongodb-org/5.0/x86_64/
+               baseurl=https://repo.mongodb.org/yum/amazon/2/mongodb-org/{+mdb-version+}/x86_64/
```

Is there a reason not to try to match the RHEL instructions w.r.t. using variables for the OS version/architecture?

```suggestion
               baseurl=https://repo.mongodb.org/yum/amazon/$releasever/mongodb-org/{+mdb-version+}/$basearch/
```

I’ve verified manually that this still works for Amazon 2 on x64, and the we do provide downloads for arm64 and Amazon Linux 2022 as well, so this seems like the most future-proof version.